cvmfs/options.cc

```diff
@@ -434,6 +457,72 @@ void OptionsManager::UnsetValue(const string &key) {
     unsetenv(key.c_str());
 }
 
+const std::string DefaultOptionsTemplateManager
```

Comment:
Ah, I forgot about this the first time: for the `const static` members, let's use only plain old data types, that is in this case `char *`. The reason is that they get constructed at some, unclear point at program start, and with more complex C++ objects being constructed at this point, funny things can happen.